packages/table/package.json

```diff
@@ -8,6 +8,8 @@
   "unpkg": "dist/table.bundle.js",
   "scripts": {
     "build:gulp": "$(cd ..; npm bin)/gulp tsc-table sass-table",
```

Comment:
if i understand your concern properly: gulp always runs with maximum parallelization (based on task deps) so we're already doing that